cardano_node_tests/utils/dbsync_utils.py

```diff
@@ -848,6 +849,168 @@ def check_param_proposal(protocol_params: dict) -> tp.Optional[dbsync_queries.Pa
+        # Economic proposals group
+        "min_fee_a": pp.get("txFeePerByte"),
+        "min_fee_b": pp.get("txFeeFixed"),
+        "key_deposit": pp.get("stakeAddressDeposit"),
+        "pool_deposit": pp.get("stakePoolDeposit"),
+        "monetary_expand_rate": _evaluate_pparam(pp.get("monetaryExpansion")),
+        "treasury_growth_rate": _evaluate_pparam(pp.get("treasuryCut")),
+        "min_pool_cost": pp.get("minPoolCost"),
+        "coins_per_utxo_size": pp.get("coinsPerUTxOByte"),
+        "min_fee_ref_script_cost_per_byte": pp.get("minFeeRefScriptCostPerByte"),
+        "price_mem": price_mem,
+        "price_step": price_steps,
+        # Technical proposals group
+        "influence": _evaluate_pparam(pp.get("poolPledgeInfluence")),
+        "max_epoch": pp.get("poolRetireMaxEpoch"),
+        "optimal_pool_count": pp.get("stakePoolTargetNum"),
+        "collateral_percent": pp.get("collateralPercentage"),
+        # Governance proposal group
+        # - DReps
+        "dvt_committee_no_confidence": _evaluate_pparam(dvt.get("committeeNoConfidence")),
+        "dvt_committee_normal": _evaluate_pparam(dvt.get("committeeNormal")),
+        "dvt_hard_fork_initiation": _evaluate_pparam(dvt.get("hardForkInitiation")),
+        "dvt_motion_no_confidence": _evaluate_pparam(dvt.get("motionNoConfidence")),
+        "dvt_p_p_economic_group": _evaluate_pparam(dvt.get("ppEconomicGroup")),
+        "dvt_p_p_gov_group": _evaluate_pparam(dvt.get("ppGovGroup")),
+        "dvt_p_p_network_group": _evaluate_pparam(dvt.get("ppNetworkGroup")),
+        "dvt_p_p_technical_group": _evaluate_pparam(dvt.get("ppTechnicalGroup")),
+        "dvt_treasury_withdrawal": _evaluate_pparam(dvt.get("treasuryWithdrawal")),
+        "dvt_update_to_constitution": _evaluate_pparam(dvt.get("updateToConstitution")),
+        # - Pools
+        "pvt_committee_no_confidence": _evaluate_pparam(pvt.get("committeeNoConfidence")),
+        "pvt_committee_normal": _evaluate_pparam(pvt.get("committeeNormal")),
+        "pvt_hard_fork_initiation": _evaluate_pparam(pvt.get("hardForkInitiation")),
+        "pvt_motion_no_confidence": _evaluate_pparam(pvt.get("motionNoConfidence")),
+        "pvtpp_security_group": _evaluate_pparam(pvt.get("ppSecurityGroup")),
+        # General
+        "gov_action_lifetime": pp.get("govActionLifetime"),
+        "gov_action_deposit": pp.get("govActionDeposit"),
+        "drep_deposit": pp.get("dRepDeposit"),
+        "drep_activity": pp.get("dRepActivity"),
+        "committee_min_size": pp.get("committeeMinSize"),
+        "committee_max_term_length": pp.get("committeeMaxTermLength"),
+    }
+
+    return params_mapping
+
+
+def _check_param_proposal(
+    param_proposal_db: tp.Union[dbsync_queries.ParamProposalDBRow, dbsync_queries.EpochParamDBRow],
+    params_map: dict,
+) -> list:
+    """Check parameter proposal against db-sync."""
+    failures = []
+
+    for param_db, protocol_value in params_map.items():
```

Comment:
Maybe `param_name` or `param_from_db` would be better. The `param_db` is bit confusing, I was expecting the data to be some kind of dict (or other kind of database) and was surprised to see it used in `getattr` below.